test-app/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@tsconfig/ember/tsconfig.json",
   "compilerOptions": {
+    "target": "es2022",
```

Comment:
I'm just wondering if this was needed for some reason?

Reply:
Yeah, right... this was needed to make TS understand that `Error#cause` is a thing. Was getting type errors without it when trying to access `.cause`.